osc/output/input.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import textwrap
+from typing import Dict
+from typing import Optional
+
+from .. import oscerr
+from .tty import colorize
+
+
+def get_user_input(question: str, answers: Dict[str, str], default_answer: Optional[str] = None):
+    """
+    Ask user a question and wait for reply.
+
+    :param question: The question.
+    :param answers: A dictionary with answers. Keys are expected replys and values are their descriptions.
+    :param default_answer: The default answer. Must be ``None`` or match an ``answers`` entry.
+    """
+
+    if default_answer and default_answer not in answers:
+        raise ValueError(f"Default answer doesn't match any answer: {default_answer}")
+
+    question = textwrap.dedent(question)
+    question = question.strip()
+
+    prompt = []
+    for key, value in answers.items():
+        value = f"{colorize(key, 'bold')}){value}"
+        prompt.append(value)
+
+    prompt_str = " / ".join(prompt)
+    if default_answer:
+        prompt_str += f" (default={colorize(default_answer, 'bold')})"
+    prompt_str += ": "
+
+    print(question, file=sys.stderr)
+
+    while True:
+        try:
+            reply = input(prompt_str)
+        except EOFError:
+            # interpret ctrl-d as user abort
+            raise oscerr.UserAbort() from None
```

Comment:
Why the `from None`?

Reply:
Well, the warning is pretty dumb imho. I'd just ignore it :shrug:

Reply:
Done. I added `# pylint: ignore=` that solves it